src/index.ts

```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
+function addComment(crawlerId: string): void {
```

Comment:
This PR should have a comment no?

Reply:
Actually, the GA tests are running the version 0.6 which doesn't have the comment feature yet. I'm not sure how to test it efficently